School/teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(age, _name, _parent_permission, specialization)
+    super(age, name: 'unknown', parent_permission: true)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Kindly make sure that the parameters passed into the constructor, get passed to the super constructor instead of passing in the default values each time. 

- [HINT] You  can use these default values on the constructor itself and pass the parameters to the super constructor like the following:
```ruby
  def initialize(age, _name = 'unknown', _parent_permission = true, specialization)
    super(age, _name, parent_permission: _parent_permission)
    @specialization = specialization
  end
```